eng/Directory.Build.Data.targets

```diff
@@ -222,4 +222,15 @@
   </Target>
 
   <Import Project="$(CentralPackageVersionPackagePath)\Sdk.targets" />
+
+  <!-- InheritDoc-->
+  <ItemGroup Condition="'$(InheritDocEnabled)' == 'true'">
+    <PackageReference Include="SauceControl.InheritDoc" Version="1.1.1" PrivateAssets="all" />
```

Comment:
This looks to be a third party library, is it safe for use here? 

Also you will need to move the version number to the Packages.Data.props file. The build should actually be failing for this.

Reply:
> This looks to be a third party library, is it safe for use here?

It's MIT license. Any concerns other than it is 3rd party?

Reply:
No particular concern I just wanted to ensure we understand what dependencies we are taking. This is just a tool and not a runtime dependency so it is less of an issue but in general we have to assume 3rd party dependencies may have issues such as no maintainer in the future.